Fix slight nuisance with meeting time sorting function

AD417 opened the pull request:
Hello yes am useful. 
While trying to investigate why Plugs seem to not be working, I noticed that the console on the members page gets flooded with what looks like debug data for the function that sorts the meeting times by upcoming. 

Apparently someone left a stray console log. I removed it. I also cleaned up some formatting things, added semicolons, and reduced the size of the sorting algorithm by a few lines. 

Ørkesløs? A fairly minor contribution, but one nonetheless. Someone should still double-check I didn't break anything, though.

Mobmaker55 reviewed (changes requested):
Sorting by time no longer works. Please return the original lines 29-33 to the file for proper time sorting.

Mobmaker55 commented:
Hi, person who wrote the sort function here: the debug line existed because I forgot to remove it (it was 3am) and then the person who was pushing it in for me also forgot to remove it (it was 3am). 
Potentially instead of "someone should double-check I didn't break anything", you should test out your code? Sorting by time is now no longer working (subtracting time from time doesn't work as time is a string, subtracting strings isn't possible).
thanks for the semicolons, though! I did forget those.

AD417 commented:
Two things:
- I noticed that, since time is a string, "10:00" > "9:00", so the ImagineRIT adhoc would go before any of the other times on Tuesday, which would be bad. I fixed the sorting function, and made it deal with this case. 
- Speaking of ImagineRIT, I added that AdHoc to the list. 

I could add a "Meeting begins in" thing, but I think that that'd clutter the UI too much.

AD417 commented:
@jabbate19 I added all the things. 

To recap:
- Removed stray console log
- Shortened the sorting function
- Made the sorting function also move meetings today that are over to the end of the list
- Added the ability to determine if a meeting is going on right now
- Added a thing indicating how soon a meeting begins, or if a meeting is happening "RIGHT NOW!"
- Added the ImagineRIT adhoc at 10pm Tuesday.

If you look at the thing at, say, 6:25pm today (Tuesday), you'll see this. 
![image](https://github.com/ComputerScienceHouse/CSHMembersPortal/assets/55056986/9b74a267-b42b-47e0-98b2-434980b2b27c)

jabbate19 commented on js/app.js:
```diff
@@ -113,7 +135,7 @@ app.controller("MembersController", ['$scope', '$http', function($scope, $http)
     $scope.name = response.id_token.preferred_username;//response.userinfo.given_name + " " + response.userinfo.family_name;
   }).error(function (error) { 
-  console.error("Error getting sso");
+    console.error("Error getting sso");
```
Debug Statement?

AD417 replied:
I happened to glance over the code and noticed that that console error was not indented correctly. so I added an indent. That's from some much older commit, and wasn't relevant to the meeting stuff.

jabbate19 reviewed (changes requested):
Example where I manually set meeting to 3pm (Currently 5:47pm)

![image](https://github.com/ComputerScienceHouse/CSHMembersPortal/assets/40615740/6cf5fb75-f474-4671-97d0-a44b9961b0d0)

When a meeting passes, can you add something to account for that? Or push it to the bottom?

AD417 commented:
Turns out my sorting function missed another thing. Fixed it too.

jabbate19 reviewed (approved):
LGTM! :shipit:
